File: 프로그래머스/연습문제/Lv1/시저 암호/시저 암호.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s, int n) {
    for(int i=0; i<s.length(); i++){
        if(s[i] == ' ') continue;
        else if(s[i] <= 'Z'){
            int a = s[i]-'A';
            a = (a+n)%26;
            s[i] = a+'A';
        }
        else{
            int a = s[i]-'a';
            a = (a+n)%26;
            s[i] = a+'a';
        }
    }
    
    return s;
}

File: 프로그래머스/연습문제/Lv1/시저 암호/시저 암호.py
def solution(s, n):
    answer = ""
    for si in s:
        if si == ' ': answer += ' '
        elif si <= 'Z':
            a = ord(si)-ord('A')
            a = (a+n)%26
            answer += chr(a+ord('A'))
        else:
            a = ord(si)-ord('a')
            a = (a+n)%26
            answer += chr(a+ord('a'))

    return answer
